models/business-unit/README.md

```diff
@@ -0,0 +1,26 @@
+# @commercetools-test-data/business-unit
+
+This package provides the data model for the commercetools platform `BusinessUnit` representations
+
+https://docs.commercetools.com/api/projects/business-units#representations
+
+# Install
+
+```bash
+$ pnpm add -D @commercetools-test-data/business-unit
+```
+
+# Usage
+
+Please take note that in comparison to previously designed packages and documentation, this format demonstrates a subtle divergence. Company and Division are both classified as distinct types of Business Units, and therefore share a common representation within the model.
```

Comment:
I probably would have not referenced "previously designed packages" because someone consuming this and reading the Readme doesn't know what came first. Just explain how and why it's different than others.